### DISCUSSION TITLE
🙏暂停时显示进度条

### DISCUSSION THREAD

**Discussion by SurferRosa** [diana7127/mpv.net-DW#36] [Q&A] [answered]
请问如何隐藏? 经常暂停不方便,会把字幕顶起来,内置的字幕就挡住了.(不是osd)

**Comment by MC-dusk** [comment]
+1

**Comment by htyxyt** [comment]
script-opts->modernx.conf里添加一句showonpause = false

**Reply by MC-dusk** [reply]
没有modernx.conf新建了一个，填进去，但是没效果。v1.6.0

**Reply by MC-dusk** [reply, marked as answer]
不过提醒了我，可以通过修改mpv.net-DW\portable_config\scripts\modernx.lua，搜索showonpause = true改成false就行了

**Comment by owlx56** [comment]
修改 `mpv.net-DW\portable_config\scripts\modernx.lua`，搜索 `showonpause = true` 将 `true` 改成 `false` 即可

@SurferRosa 麻烦采纳，让更多人看到
